app/controllers/fields_of_studies_controller.ts

```diff
@@ -0,0 +1,36 @@
+import type { HttpContext } from "@adonisjs/core/http";
+
+import FieldsOfStudy from "#models/field_of_study";
+import { paginationValidator } from "#validators/pagination";
+import { showValidator } from "#validators/show";
+
+export default class FieldsOfStudiesController {
+  async index({ request }: HttpContext) {
+    const { page, limit } = await request.validateUsing(paginationValidator);
+    const baseQuery = FieldsOfStudy.query().withScopes((scopes) => {
+      scopes.preloadRelations(request.only(["departments"]));
+      scopes.handleSortQuery(request.input("sort"));
+    });
```

Comment:
Similar issues as in the departments controller:
- `departments` is not a valid relation here, `department` is.
  - move the list of supported relations into a readonly class field to reduce code duplication
- surely we'd want to support filtering on this endpoint?